spec/system/managing_topics_spec.rb

```diff
@@ -21,4 +21,22 @@
     expect(current_path).to eq topics_path
     expect(page).to have_content 'tell us a story'
   end
+
+  scenario 'viewing a topic' do
+    visit topics_path
+    click_on('What is your favourite colour')
+    expect(current_path).to eq topic_path(@topic.id)
+    expect(page).to have_content('Topic: 1')
+  end
+
+  scenario 'when editing a topic' do
+    visit topic_path(@topic.id)
+    click_link 'Edit Topic'
+    expect(current_path).to eq edit_topic_path(@topic.id)
+    expect(page).to have_content 'What is your favourite colour'
+    fill_in 'Name', with: 'this is my new topic'
```

Comment:
What about trying out an error (e.g. submitting with a blank name), then you can see if you get the validation error that you expect. (Hint: it's not working at present)